### PR TITLE
save modifiers in a private dict in the config

### PR DESCRIPTION
**Pull request by RokasEl**
this allows for faster loading as the modifications don't need to be initialised each time
in addition, the kwargs now contains the current keys-values from the live settings of the modification

finally, changed the buffer size which was causing weird behaviour in terms of things being saved in atoms.info
and could cause endless streaming of the frames

### CODE REVIEW

**Comment by RokasEl** [comment]
@PythonFZ, this probably still needs the button to purge the loaded modifications. Although, I don't think the modification objects take up that much space and doubt there would be a use case where someone loads up so many modifiers that it becomes a problem.

**Comment by PythonFZ** [comment]
> @PythonFZ, this probably still needs the button to purge the loaded modifications. Although, I don't think the modification objects take up that much space and doubt there would be a use case where someone loads up so many modifiers that it becomes a problem.

Do you think there could be an issue e.g. with GPU memory allocation?
Should we just do `_loaded_modifiers={}` or call `del` on all instances stored in the `_loaded_modifiers`?

**Comment by RokasEl** [comment]
I think calling `del` is safer with respect to garbage collection. 

>Do you think there could be an issue e.g. with GPU memory allocation?
I don't think we might run out of GPU memory if we try to load to many things but I don't think we'll get errors where two objects try to allocate to the same GPU memory.

File: zndraw/shared.py [hunk 1]
```diff
@@ -58,10 +58,10 @@ class Config(BaseModel):
     selection_functions: typing.List[str] = _SELECTION_FUNCTIONS
     bonds_functions: typing.List[str] = _BONDS_FUNCTIONS
     js_frame_buffer: tuple = Field(
-        (50, 200), description="Javascript frame buffer in negative/positive direction"
+        (2, 2), description="Javascript frame buffer in negative/positive direction"
```
[the remark this answers is not in the document]
Comment (RokasEl) [reply zndraw/shared.py:61]: 50, 50 works on my system and feels quite responsive
```suggestion
    js_frame_buffer: tuple = Field(
        (50, 50), description="Javascript frame buffer in negative/positive direction"
```

**Comment by RokasEl** [comment]
I think we can merge this now.

**Pull request merged**